File: DailyChallenge/2020/March/3.py
# https://leetcode.com/explore/challenge/card/march-leetcoding-challenge-2021/588/week-1-march-1st-march-7th/3659/

class Solution(object):
    def missingNumber(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        s = sum(nums)
        n = len(nums)
        S = int((n*(n+1))/2)
        return S-s

File: DailyChallenge/2020/May/3.java
// https://leetcode.com/explore/challenge/card/may-leetcoding-challenge-2021/598/week-1-may-1st-may-7th/3730/

class Solution {
    public int[] runningSum(int[] nums) {
        int len = nums.length;
        int[] ans = new int[len];
        ans[0] = nums[0];
        for(int i=1;i<len;i++){
            ans[i] = ans[i-1] + nums[i];
        }
        return ans;
    }
}
